src/installer.ts

```diff
@@ -128,15 +128,18 @@ function updateStatusBarItem(sbItem : vscode.StatusBarItem, text: string, toolti
 }
 
 export async function installKamel(context: vscode.ExtensionContext): Promise<Errorable<null>> {
-	const latestversion = await versionUtils.getLatestCamelKVersion();
-	if (failed(latestversion)) {
-		return { succeeded: false, error: latestversion.error };
-	}
-
-	let versionToUse = latestversion.result.trim();
+	let versionToUse: string;
 	let runtimeVersionSetting = vscode.workspace.getConfiguration().get(config.RUNTIME_VERSION_KEY) as string;
```

Comment:
if runtimeVersionSetting is never changed again then make it a const instead of a let

Reply:
I know its not part of your changes but I wanted to mention it anyways.

Reply:
updated as it is a trivial change